openmc/filter.py

```diff
@@ -77,6 +77,23 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def __gt__(self, other):
+        if self.type != other.type:
+            if self.type in _FILTER_TYPES and other.type in _FILTER_TYPES:
+                delta = _FILTER_TYPES.index(self.type) - \
+                        _FILTER_TYPES.index(other.type)
+                return True if delta > 0 else False
+            else:
+                return False
+        else:
+            if 'energy' in self.type and 'energy' in other.type:
+                return self.bins[0] >= other.bins[-1]
+            else:
+                return max(self.bins) > max(other.bins)
+
+    def __lt__(self, other):
+        return not self > other
```

Comment:
What do you mean? The built-in `__le__` method should make use of the customized `__lt__` and `__eq__` methods.